web/modules/custom/joinup_community_content/src/Controller/CommunityContentController.php

```diff
@@ -97,6 +97,10 @@ public function createAccess(RdfInterface $rdf_entity, ?AccountInterface $accoun
       return AccessResult::forbidden();
     }
 
+    // @todo This is probably related to ISAICP-6007. The 'create aceess' check
+    // shouldn't require an entity. The entity is created only after the 'create
+    // access' is checked. Refactor this in ISAICP-6007.
+    // @see https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-6007
```

Comment:
I think this was done so that we can have our access controller much more compact. In order to determine if the user can create the content, we need to also know the group and the default state apart from the default parameter, being the bundle.

Reply:
Well, let's keep the todo there. That can be fixed w/o needing to create an entity.